src/frontend/Sider/Behandling/Inngangsvilkår/Målgruppe/EndreMålgruppeRad.tsx

```diff
@@ -115,15 +115,18 @@ const EndreMålgruppeRad: React.FC<{
             }
             feilmelding={feilmelding}
         >
-            <MålgruppeVilkår
-                målgruppeForm={målgruppeForm}
-                oppdaterDelvilkår={(key: keyof DelvilkårMålgruppe, vurdering: Vurdering) =>
-                    settMålgruppeForm((prevState) => ({
-                        ...prevState,
-                        delvilkår: { ...prevState.delvilkår, [key]: vurdering },
-                    }))
-                }
-            />
+            {målgruppeForm.type !== '' && (
```

Comment:
Sjekken på om Vilkår skal vises eller ikke gjøres på 2 ulike måter
Her sjekkes typen for verdi utenfor komponenten, mens det i prinsipp gjøres inne i komponenten på AktivitetVilkår.
Tenker det kan være fint å gjøre det på lik måte, og kan kanskje gjøre det inne i Vilkår-komponenten for å unngå verdi-sjekken i Endre*Rad-fila?